src/virtual-machine.js

```diff
@@ -434,7 +435,11 @@ class VirtualMachine extends EventEmitter {
                 return resolve();
             }
 
-            resolve(this.loadProject(await response.text()));
+            const arrayBuffer = await response.arrayBuffer();
```

Comment:
Just real quick wanna verify this is necessary, as I'm not sure why this is NOW necessary. Will report back

Reply:
see above